index.js

```diff
@@ -67,13 +67,29 @@ class ExecutorRouter extends Executor {
      * @param {String} config.token         JWT to act on behalf of the build
      * @return {Promise}
      */
-    start(config) {
+    _start(config) {
```

Comment:
Here is the base class this one extends: https://github.com/screwdriver-cd/executor-base